src/shared/tpm2-util.c

```diff
@@ -2704,8 +2704,10 @@ int tpm2_get_best_pcr_bank(
                         return r;
 
                 /* If variable is not set use guesswork below */
-                log_debug("Boot loader didn't set LoaderTpm2ActivePcrBanks EFI variable, we have to guess used PCR banks.");
-        } else {
+                log_debug("Boot loader didn't set the LoaderTpm2ActivePcrBanks EFI variable, we have to guess the used PCR banks.");
```

Comment:
Not a big matter, but the insertion of `the` should be moved to the previous commit.